onmt/inputters/corpus.py

```diff
@@ -387,7 +423,7 @@ def save_transformed_sample(opts, transforms, n_sample=3):
                 maybe_example = DatasetAdapter._process(item, is_train=True)
                 if maybe_example is None:
                     continue
-                src_line, tgt_line = maybe_example['src'], maybe_example['tgt']
+                src_line, tgt_line = maybe_example['src']['src'], maybe_example['tgt']['tgt']
```

Comment:
This is not very code reader friendly. Maybe put a comment to remind of the structure of the fields?